homeassistant/components/recorder/statistics.py

```diff
@@ -1485,16 +1485,13 @@ def statistic_during_period(
 
         # Calculate the tail period
         tail_start_time: datetime | None = None
-        tail_end_time: datetime | None = None
+        tail_end_time: datetime | None = end_time
```

Comment:
Is this really correct, shouldn't we instead set `tail_end_time` to `end_time` in the two cases below?